<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OtobusBilgisiVeriGirisFormu</class>
 <widget class="QWidget" name="OtobusBilgisiVeriGirisFormu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>318</width>
    <height>213</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Otobus Bilgisi</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../Kaynaklar.qrc">
    <normaloff>:/Resimler/Resimler/otobus.png</normaloff>:/Resimler/Resimler/otobus.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <widget class="QLabel" name="lbIcon">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../../Kaynaklar.qrc">:/Resimler/Resimler/otobus.png</pixmap>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="lbOtobusBilgisi">
       <property name="text">
        <string>Otobüs Bilgisi</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="4" column="0">
          <widget class="QLabel" name="lbRenk">
           <property name="text">
            <string>Renk</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="leKoltukSayisi"/>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lePlaka"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="lbKoltukSayisi">
           <property name="text">
            <string>Koltuk Sayısı</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QDateEdit" name="deMuayeneTarihi"/>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="leRuhsat"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="lbMuayeneTarihi">
           <property name="text">
            <string>Muayene Tarihi</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="lbRuhsat">
           <property name="text">
            <string>Ruhsat</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="leRenk"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="lbPlaka">
           <property name="text">
            <string>Plaka</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QPushButton" name="pbKaydet">
           <property name="text">
            <string>Kaydet</string>
           </property>
           <property name="icon">
            <iconset resource="../../Kaynaklar.qrc">
             <normaloff>:/Resimler/Resimler/kaydet.png</normaloff>:/Resimler/Resimler/kaydet.png</iconset>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="pbIptal">
           <property name="text">
            <string>İptal</string>
           </property>
           <property name="icon">
            <iconset resource="../../Kaynaklar.qrc">
             <normaloff>:/Resimler/Resimler/iptal.png</normaloff>:/Resimler/Resimler/iptal.png</iconset>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lePlaka</tabstop>
  <tabstop>leKoltukSayisi</tabstop>
  <tabstop>leRuhsat</tabstop>
  <tabstop>deMuayeneTarihi</tabstop>
  <tabstop>leRenk</tabstop>
  <tabstop>pbKaydet</tabstop>
  <tabstop>pbIptal</tabstop>
 </tabstops>
 <resources>
  <include location="../../Kaynaklar.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pbIptal</sender>
   <signal>clicked()</signal>
   <receiver>OtobusBilgisiVeriGirisFormu</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>272</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>310</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
